platformio.ini

```diff
@@ -15,10 +15,10 @@ build_flags =
 	${env.build_flags}
 	-DMONITOR_CLASS=DisplayMonitor
 	-DBLINK_LED=2
-	-DGATEWAY_SSID='"My GW Network (change me)"'
-	-DGATEWAY_SSID_PASSWORD='"my p4ssw0rd! (change me)"'
-upload_port = COM5
-monitor_port = COM5
+	-DGATEWAY_SSID='"NET_2G45C3B2"'
+	-DGATEWAY_SSID_PASSWORD='"B345C3B2"'
```

Comment:
Is this your home network SSID and password?

Reply:
We already corret this.
https://github.com/Instituto-Atlantico/mesh-connect/blob/ivo/transmission80211wifinode/platformio.ini